app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class StaticPagesController < ApplicationController
```

Comment:
I added this because `devise` needed a `root` to connect to. I put `static` controller/view as a stand-in.